ZuluContent/Misc/Titles.cs

```diff
@@ -4,354 +4,372 @@
 
 namespace Server.Misc
 {
-  public class Titles
-  {
-    public const int MinFame = 0;
-    public const int MaxFame = 15000;
-
-    public static void AwardFame(Mobile m, int offset, bool message)
+    public class Titles
     {
-      if (offset > 0)
-      {
-        if (m.Fame >= MaxFame)
-          return;
-
-        offset -= m.Fame / 100;
-
-        if (offset < 0)
-          offset = 0;
-      }
-      else if (offset < 0)
-      {
-        if (m.Fame <= MinFame)
-          return;
-
-        offset -= m.Fame / 100;
-
-        if (offset > 0)
-          offset = 0;
-      }
-
-      if (m.Fame + offset > MaxFame)
-        offset = MaxFame - m.Fame;
-      else if (m.Fame + offset < MinFame)
-        offset = MinFame - m.Fame;
-
-      m.Fame += offset;
-
-      if (message)
-      {
-        if (offset > 40)
-          m.SendLocalizedMessage(1019054); // You have gained a lot of fame.
-        else if (offset > 20)
-          m.SendLocalizedMessage(1019053); // You have gained a good amount of fame.
-        else if (offset > 10)
-          m.SendLocalizedMessage(1019052); // You have gained some fame.
-        else if (offset > 0)
-          m.SendLocalizedMessage(1019051); // You have gained a little fame.
-        else if (offset < -40)
-          m.SendLocalizedMessage(1019058); // You have lost a lot of fame.
-        else if (offset < -20)
-          m.SendLocalizedMessage(1019057); // You have lost a good amount of fame.
-        else if (offset < -10)
-          m.SendLocalizedMessage(1019056); // You have lost some fame.
-        else if (offset < 0)
-          m.SendLocalizedMessage(1019055); // You have lost a little fame.
-      }
-    }
+        public const int MinFame = 0;
+        public const int MaxFame = 15000;
 
-    public const int MinKarma = -15000;
-    public const int MaxKarma = 15000;
+        public const int KarmaCriminalLimit = -625;
```

Comment:
Only new addition here, we need to make a separate issue to implement ZH3 karmafame

Reply:
Cool, should we put this in a config file tho?

Reply:
Yeah this is just a placeholder, when we do the [karma fame implementation](https://github.com/zuluhotelaustralia/zha-2004-2006-side/blob/main/zhc93/pkg/opt/karmafame/karmafame.inc#L5-L43) we should load all the constants from JSON files. But that's outside the scope of this PR.

Reply:
ok